app/models/world_wide_taxon.rb

```diff
@@ -58,7 +58,7 @@ def merge(to_merge)
 
   def can_subscribe?
     # Is @can_subscribe ever used on world wide pages?
-    return @can_subscribe if defined?(@can_subscribe)
+    return can_subscribe if defined?(can_subscribe)
```

Comment:
Not sure about this as it changes the result of `if defined?(...)` because the method `can_subscribe` is always defined, whereas the instance var may or may not be defined. Very hard to understand what this even does though. I've searched the usage in the master branch I don't see that instance var set anywhere...